README.md

```
@@ -1,81 +1,233 @@
+<br>
+<img src="https://img.shields.io/badge/git-F05032?style=for-the-badge&logo=git&logoColor=white"> <img src="https://img.shields.io/badge/github-181717?style=for-the-badge&logo=github&logoColor=white">
+</div>
+
+<br/>
+
+# 🗂️ 디렉토리 구조
+
+모노레포(터보레포)
 
-This is an official starter Turborepo.
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)